src/middlewared/middlewared/schema.py

```diff
@@ -172,9 +172,9 @@ def validate(self, value):
 
 class IPAddr(Str):
 
-    def validate(self, value):
+    def validate(self, value, cidr=True):
         if value:
-            value = value.rsplit('/')[0]  # May have CIDR
+            value = value.rsplit('/')[0] if cidr else value
```

Comment:
Again, we should check that `value` is `192.168.0.0/24` and not `192.168.0.0/24/garbage`